.github/workflows/conda_ci.yml

```diff
@@ -0,0 +1,60 @@
+name: Conda CI Tests
+
+on:
+  # Run if anything changes related to tool setup
+  push:
+    paths: 'setup/**'
+  pull_request:
+    paths: 'setup/**'
+  workflow_dispatch:
+    
+jobs:
+  test-basic-tools:
+    name: Test basic tools
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
+    defaults:
+      run:
+        shell: bash -el {0}
+    steps:
+      - name: Checkout siliconcompiler
+        uses: actions/checkout@v4
+        
+      - name: Setup conda env
+        uses: conda-incubator/setup-miniconda@v3
+        with:
+          activate-environment: sc_env
+          environment-file: setup/conda/environment.yml
+        
+      - name: Tool version check
+        run : |
+          conda activate sc_env
+
+          klayout -v
+          openroad -version
+          surelog --version
+          yosys --version
+          
+      - name: Set up Python ${{ matrix.python-version }}
+        id: setup-python
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: 'pip'
+          allow-prereleases: true
+
+      - name: Run tests
+        run : |
+          conda activate sc_env
+          
+          ${{ steps.setup-python.outputs.python-path }} -m venv clean_env
+          source clean_env/bin/activate
+          python3 --version
+          
+          cd $GITHUB_WORKSPACE
+          python3 -m pip install --upgrade pip
+          python3 -m pip install .[test]
+          pytest -n auto -k "not sim" tests/tools/test_openroad.py tests/tools/test_klayout.py tests/tools/test_surelog.py tests/tools/test_yosys.py tests/core/test_tool_tasks.py tests/flows/test_show.py tests/apps/test_sc_show.py tests/examples/test_heartbeat.py
```

Comment:
sim is not a pytest marker we use, what is it for?

Reply:
The `test_heartbeat.v` contains the "test_sim" test which can't be run because we haven't added icarus to the conda setup. The `-k "not sim"` should filter that test out by name

Reply:
ahh, I see I missed the `-k` vs `-m`.